testplan/common/utils/match.py

```diff
@@ -147,27 +151,27 @@ def match(self, regex, timeout=5):
                     match = regex.match(line)
                     if match:
                         break
-                else:
+                elif timeout:
```

Comment:
I seem to be missing where `timeout` is being used other than truthy/falsy, yet in the `raise` below it seems to be made explicit we use it. Can you please pinpoint me?

Reply:
Ah it's indirectly used in `end_time`, which is used on line 156.

Reply:
Nice, I see, just had to expand the lines from front. Thanks, it looks good this way.